apps/staking/components/DevSheet.tsx

```diff
@@ -207,67 +201,68 @@ function FeatureFlagToggle({
   );
 }
 
-function ContractActions() {
-  const [value, setValue] = useState<string>('0');
-  const serviceNodeRewardsAddress = addresses.ServiceNodeRewards.testnet;
-
-  const tokenAmount = useMemo(() => BigInt(value) * BigInt(10 ** SENT_DECIMALS), [value]);
-
-  const {
-    allowance,
-    refetch,
-    status: allowanceStatus,
-  } = useAllowanceQuery({
-    contractAddress: serviceNodeRewardsAddress,
-  });
-
-  const { approveWrite, resetApprove, status } = useProxyApproval({
-    contractAddress: serviceNodeRewardsAddress,
-    tokenAmount,
-  });
-
-  const handleClick = () => {
-    if (status !== 'idle') {
-      resetApprove();
-    }
-    approveWrite();
-  };
-
-  useEffect(() => {
-    if (status === 'success') refetch();
-  }, [status]);
-
-  return (
-    <>
-      <SheetTitle>Contract Actions 🚀</SheetTitle>
-      <span className="inline-flex justify-start gap-1 align-middle">
-        <span className="inline-flex justify-start gap-1 align-middle">
-          {'Allowance:'}
-          <span className="text-session-green">
-            {allowanceStatus === 'success' ? formatSENTBigInt(allowance) : <LoadingText />}
-          </span>
-        </span>
-      </span>
-      <Input type="number" value={value} onChange={(e) => setValue(e.target.value)} />
-      <Button
-        data-testid="button:reset-allowance"
-        onClick={handleClick}
-        size="sm"
-        rounded="md"
-        disabled={status === 'pending' || tokenAmount === allowance}
-      >
-        {status === 'pending' ? (
-          <LoadingText />
-        ) : tokenAmount > BigInt(0) ? (
-          'Set Allowance'
-        ) : (
-          'Reset Allowance'
-        )}
-      </Button>
-      <ExitNodes />
-    </>
-  );
-}
+//
+// function ContractActions() {
```

Comment:
Maybe put this into a separate file if you aren't going to use it again anytime soon?

Reply:
I will, also dont want to spread dev sheet out more, its important its not imported when in prod